packages/kernel-6.12/kernel-6.12.spec

```
@@ -82,6 +82,9 @@ Conflicts: %{_cross_os}image-feature(external-kmod-development)
 # Pull in expected modules.
 Requires: %{name}-modules = %{version}-%{release}
 
+# Pull in default mkfs conf for xfsprogs.
+Requires: (%{name}(mkfs-xfs-conf) if %{_cross_os}xfsprogs)
```

No need to bother with these capabilities:
```suggestion
Requires: (%{name}-mkfs-xfs-conf if %{_cross_os}xfsprogs)
```